### ISSUE TITLE
Implementetion of EspFileManager for LittleFS

### ISSUE THREAD

**Issue by forty76** [arslan437/EspFileManager#1]
Hi, I want ask you to implement library with LittleFS

I have just change your code and all work fine. 

void EspFileManager::FSlistDir(const char * dirname, uint8_t levels)
{
    DEBUGX("Listing directory: %s\n", dirname);

    File root = LittleFS.open(dirname);
    if(!root){
        DEBUGLF("Failed to open directory");
        return;
    }
    if(!root.isDirectory()){
        DEBUGLF("Not a directory");
        return;
    }

    bool first_files = true;
    str_dataFS = "";
    File file = root.openNextFile();
    while(file){
        if (first_files)
            first_files = false;
        else 
            str_dataFS += ":";

        if(file.isDirectory()){
            // DEBUGF("  DIR : ");
            // DEBUGL(file.name());
            str_dataFS += "1,";
            str_dataFS += file.name();
            // if(levels){
            //     listDir(file.path(), levels -1);
            // }
        } else {
            // DEBUG("  FILE: ");
            // DEBUG(file.name());
            // DEBUG("  SIZE: ");
            // DEBUGL(file.size());
            str_dataFS += "0,";
            str_dataFS += file.name();
        }
        file = root.openNextFile();
    }

    file.close();
    // DEBUGL2("Folder string ", str_data);
}

**Comment by arslan437** [comment]
OK, 

I am also planning the support for the little FS, If I find some time. I will add that to the library.

**Comment by forty76** [comment]
In upload section you don' t use right destination path. 

    server->on("/upload", HTTP_POST, [&](AsyncWebServerRequest *request)
        { request->send(200, "application/json", "{\"status\":\"success\",\"message\":\"File upload complete\"}"); }, [&](AsyncWebServerRequest *request, String filename, size_t index, uint8_t *data, size_t len, bool final){
        String file_path;

        file_path = "/";
        file_path += filename;

variable filename have file name only, so file is uploaded in root path. 
In delete section 
path = request->getParam("path")->value();  give file_name + path

**Comment by arslan437** [comment]
Yes, in the current version the uploaded file will go the root folder. I will update so, that the file will be uploaded to the opened folder on the website
